### PR TITLE
implements JSON Mappings

### PR DESCRIPTION
**Pull request by data-niklas**
Just adds JSON mappings, so that RSS Items can easily be converted to a JSON String.

### CODE REVIEW

File: src/rss.cr [hunk 1]
```diff
@@ -7,6 +7,17 @@ module RSS
   class Item
     property title, link, pubDate, comments, description, guid, author, category
 
+    JSON.mapping({
```
Comment (ruivieira) [inline src/rss.cr:11]: The `json` module must be required in order to use the JSON mapping macro.

**Review by ruivieira — CHANGES_REQUESTED** [review]
Hi @data-niklas, thanks for the PR!

Looks good to me, just a couple of comments:

1. You need to require the `json` module for this to work
2. If you want, please add yourself as a contributor on the main README.md

**Comment by data-niklas** [comment]
Added myself as a contributor and required the `json` module

**Review by ruivieira — APPROVED** [review]

**Comment by ruivieira** [comment]
Thanks @data-niklas!

**Pull request merged**